docs/design/3_2_multirotor_v2.tex

```diff
@@ -1,26 +1,231 @@
+
+\paragraph{Customized Add-ons}
+% TODO: add CAD parts
+
 \subsubsection{Propulsion}
+
+The propulsion system covers the majority of the multirotor parts list and determines the flight performance of the multirotor. We refer to this subsystem as the ``Drive".
+
 \paragraph{Motor Type}
+
+\underline{\textit{Brushed or brushless?}}
+Multirotors are electrically powered by a battery --- a DC source. Two common types of DC motors are  brushed motors and brushless DC (BLDC) motors. Brushed motors commutate mechanically using brushes whereas brushless motors commutate electrically using electronic controllers that switches phases in the stator coils.
+
+We use  BLDC motors for their extended usable lifespan and efficiency. Brushed motors are not appropriate for our applications for its excessive noise, friction, spark, and heat.
+
+\underline{\textit{Inrunner or outrunners?}}
+
+BLDC motors further consists of two variants: \textit{inrunner} and \textit{outrunner}. 
+
+We choose outrunner BLDC motors for its higher torque output at lower speed  (i.e. outrunner motors have low KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). The lower speed of the rotor is beneficial for two reasons. One, a motor running at a lower speed is more efficient and produces less high frequency vibrations. Two, the attached propeller can be much larger and cut through the air at a lower speed, since drag forces acting on the blades are proportional to the velocity squared. Outrunner motors do require spin-up time as the rotor is more massive and has more inertia, but this compromise is acceptable as the main operating mode of the multirotor is hovering, which requires little agility. 
+
+The alternatives are inrunner motors. Inrunner motors have the rotor on the inside and outrunner motors have the rotor on the outside. Inrunner rotors are lighter and spin faster with a lower torque which means additional gearboxes are required to extract useful torque\cite{invsoutrunner}.
+
+\paragraph{Motor Size and Speed}\label{section:motor-speed}
+
+Commercial outrunner BLDC motors generally have the following design parameters: stator size, KV/KT constants, voltage, maximum rated power, and weight. Please refer to Appendix~\ref{appendix:droneengine} for the relationship between the aforementioned design parameters. 
+
+We choose 700 KV motors with stator size 3508.  According to drone hobbist online sources (https://quadquestions.com/blog/2017/02/22/choose-right-size-motors-drone/, https://oscarliang.com/quadcopter-motor-propeller/), the ideal KV motors to maximize performance for frame sizes larger than 450 mm should be 1200 KV or less. The argument of choosing a lower KV motor follows the point made above to balance torque output and energy loss due to air friction.
```

Comment:
Ensure these are converted to proper citations

Reply:
yeah, they're "TODO's"